.gitmodules

```diff
@@ -2,5 +2,6 @@
 	path = lib/ds-test
 	url = https://github.com/dapphub/ds-test
 [submodule "lib/forge-std"]
+	branch = v1
 	path = lib/forge-std
 	url = https://github.com/brockelmore/forge-std
```

Comment:
If we're updating deps, we should also remove the explicit ds-test dep and update the forge-std URL, since this setup is a bit outdated

Reply:
Great idea! Just did it in the latest commit.